azurelinuxagent/ga/monitor.py

```diff
@@ -300,15 +306,35 @@ def reset_loggers(self):
                 self.last_reset_loggers_time = time_now
 
     def add_sysinfo(self, event):
+        """
+        This method is called after parsing the event file in the events folder and before emitting it. This means
+        all events, either coming from the agent or from the extensions, are passed through this method. The purpose
+        is to add a static list of sys_info parameters such as VMName, Region, RAM, etc. If the sys_info parameters
+        are already populated in the event, they are not overwritten.
```

Comment:
Shouldn't it be - `they ARE overwritten`?

Reply:
Thanks, it is the opposite of what I wrote. I'll update in the next iteration.